<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>458</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>533</width>
    <height>458</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>533</width>
    <height>458</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>533</width>
    <height>458</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>nRF52-Flasher</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>533</width>
     <height>458</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>533</width>
     <height>458</height>
    </size>
   </property>
   <widget class="QPushButton" name="Btn_SD">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>50</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>SoftDevice</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="File_SD">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>401</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_erase">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Erase All</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_flash">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>430</y>
      <width>211</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Flash</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/images/upload.png</normaloff>:/images/upload.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_APP">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>80</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Application</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_BOOT">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>110</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Bootloader</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="File_APP">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>401</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="File_BOOT">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>401</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_halt">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Halt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_reset">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_run">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="log">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>511</width>
      <height>261</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="console">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>441</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_send">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>400</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>150</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/images/logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>0</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Developed by: Silivanov D.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>d.silivanov@vega-absolute.ru</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>https://github.com/SmartTech/nRF52-Flasher</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>430</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>v.1.0</string>
    </property>
   </widget>
   <zorder>log</zorder>
   <zorder>Btn_SD</zorder>
   <zorder>File_SD</zorder>
   <zorder>Btn_erase</zorder>
   <zorder>Btn_flash</zorder>
   <zorder>Btn_APP</zorder>
   <zorder>Btn_BOOT</zorder>
   <zorder>File_APP</zorder>
   <zorder>File_BOOT</zorder>
   <zorder>btn_halt</zorder>
   <zorder>btn_reset</zorder>
   <zorder>btn_run</zorder>
   <zorder>console</zorder>
   <zorder>Btn_send</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
